### PR TITLE
Homework for Lesson 9

### PR DESCRIPTION
**Pull request by filkodaria**
Homework_9: task_3.

### CODE REVIEW

File: hw_9/hw_9.3/class_Team.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+		return this.teammates.length;
+	}
+
+	addTeammate(newEmployee) {
+		if (!(newEmployee instanceof ITSpecialist)) throw new Error (`Function passes only instance of class ITSpecialist.`);
+		this.teammates.push(newEmployee);
+	}
+
+	removeTeammate(name, jobTitle) {
+		const index = this.teammates.findIndex(employee => employee.name === name && employee.getSpesialization() === jobTitle);
```
Comment (anatoly-karpovich) [inline hw_9/hw_9.3/class_Team.js]: getSpesialization - no such method in ITSpecialist class

File: hw_9/hw_9.3/class_Team.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		if (index === -1) throw new Error(`Employee not found.`);
+		this.teammates.splice(index, 1);
+	}
+
+	editTeammate() {
+
```
Comment (anatoly-karpovich) [inline hw_9/hw_9.3/class_Team.js:70]: No implementation

File: hw_9/hw_9.3/class_Team.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	removeTask(id, keyWord) {
+		const regex = new RegExp(`${keyWord}`, 'gi');
+
+		const index = this.tasks.findIndex(task => task.getId() === id && task.getFeatureName().match(regex));
+		if (index === -1) throw new Error(`Task not found.`);
+		
+		this.tasks.splice(index, 1);
+	}
+
+	editTask() {
```
Comment (anatoly-karpovich) [inline hw_9/hw_9.3/class_Team.js]: Implement

File: hw_9/hw_9.3/developer.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	constructor (name, grade, experienceYears, age, country, salary, isWritingUnitTests) {
+		super(name, grade, experienceYears, age, country, salary);
+		this.isWritingUnitTests = isWritingUnitTests;
+	}
+
+	checkIsAQA() {
+		this.isWritingUnitTests ? `Dev writes Unit Tests.` : `Dev isn't able to write Unit Tests.` 
+	}
+
+	getSpesialization() {
```
Comment (anatoly-karpovich) [inline hw_9/hw_9.3/developer.js]: Should be created in ITSpecialist class first

File: hw_9/hw_9.3/developer.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+const ITSpecialist = require('./class_ITSpecialist.js');
+
+class Developer extends ITSpecialist {
+	constructor (name, grade, experienceYears, age, country, salary, isWritingUnitTests) {
+		super(name, grade, experienceYears, age, country, salary);
+		this.isWritingUnitTests = isWritingUnitTests;
+	}
+
+	checkIsAQA() {
```
Comment (anatoly-karpovich) [inline hw_9/hw_9.3/developer.js]: Seems like it's QA's method

**Review by anatoly-karpovich — APPROVED** [review]

**Comment by filkodaria** [comment]
@anatoly-karpovich 
Please, review changes)

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

**Comment by filkodaria** [comment]
Thanks a lot for your reviews!

**Pull request merged**